src/effects/effectsmanager.cpp

```diff
@@ -311,6 +318,22 @@ EffectButtonParameterSlotPointer EffectsManager::getEffectButtonParameterSlot(
     return pParameterSlot;
 }
 
+void EffectsManager::setEffectVisibility(EffectManifestPointer pManifest, bool visible) {
+    if (visible && !m_visibleEffectManifests.contains(pManifest)) {
+        auto insertion_point = std::lower_bound(m_visibleEffectManifests.begin(),
+                                                m_visibleEffectManifests.end(),
+                                                pManifest, alphabetizeEffectManifests);
+        m_visibleEffectManifests.insert(insertion_point, pManifest);
+        emit visibleEffectsUpdated();
```

Comment:
Hmm, I thought you needed parentheses around the signal name: `emit(visibleEffectsUpdated())`